### DISCUSSION TITLE
[A-L] (2023/24) Foglio 13 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#279] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/72acc43c-a574-4dbf-a3ae-91d91eb0e6fe)

**Comment by Elia-Belli** [comment]
- Espressione di $L_A$:

$$L_A\begin{pmatrix}x_1\\
x_2\\
x_3\end{pmatrix}=\begin{pmatrix}x_1-x_2+x_3\\
2x_1-3x_3\\
-2x_2+5x_3\end{pmatrix}$$

- Per determinare la dimensione del nucleo risolviamo il sistema omogeneo:

$$\begin{cases}x_1-x_2+x_3=0\\
2x_1-3x_3=0\\
-2x_2+5x_3=0\end{cases}\rightarrow^{Gauss}\begin{pmatrix}1 & -1& 1\\
0 & 2 & -5\\
0 & 0 & 0\end{pmatrix}\rightarrow\begin{cases}x_1=\frac{3}{2}x_3\\
x_2=\frac{5}{2}x_3
\end{cases}\Rightarrow Ker\ L_A = Span((\frac{3}{2},\frac{5}{2},1))\Rightarrow dim\ (Ker\ L_A)=1$$

- Sappiamo che $Im\ L_A= Span(A^1,A^2,A^3)$,
  ma per il Teorema dell Dimensione: $dim\ \mathbb{R}^3=dim(Ker L_A)+dim(Im\ L_A)\Rightarrow dim(Im\ L_A)=2$, quindi bisogna rimuovere una colonna per ottenere una base $\Rightarrow \\{A^1,A^2\\}=\\{(1,2,0),(-1,0,-2)\\}$ è una base dell'immagine.

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cfefa182-8841-40c3-b035-e62a718b79bb)

**Comment by daromacarlo** [comment]
[esercizio 2.pdf](https://github.com/sapienzastudentsnetwork/algebra2324/files/13549787/esercizio.2.pdf)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/bf661871-65fa-4b72-9a15-9c21f05e48fb)
